ci/meta.yml

```diff
@@ -196,19 +195,22 @@ meta:
               source /assume-role
               MAX_RETRIES=60
 
-
               TRUNCATE_TABLE_FUNCTION_NAME=$(jq -r '.truncate_table_lambda.value.function_name' < ./terraform-output/outputs.json)
 
-              aws ecs update-service --cluster $CLUSTER --service $SERVICE --desired-count 0 > /dev/null
-              running_count=$(aws ecs describe-services --cluster $CLUSTER --services $SERVICE | jq -r '.services[0].runningCount')
+              aws ecs update-service --cluster $CLUSTER --service azkaban-executor --desired-count 0 > /dev/null
+              aws ecs update-service --cluster $CLUSTER --service azkaban-webserver --desired-count 0 > /dev/null
+              executor_running_count=$(aws ecs describe-services --cluster $CLUSTER --services azkaban-executor | jq -r '.services[0].runningCount')
+              webserver_running_count=$(aws ecs describe-services --cluster $CLUSTER --services azkaban-webserver | jq -r '.services[0].runningCount')
 
               retries=0
-              while [ "$running_count" -ne 0 ]; do
+              while [ $executor_running_count -ne 0 ] || [ $webserver_running_count -ne 0 ]; do
                 if [ "$retries" -eq $MAX_RETRIES ]; then
-                  exit 1;
+                  echo "max retries, executor and webserver not shut down."
+                  exit 0;
```

Comment:
shouldn't this be `exit 1`? Wouldn't you want this to be a failed job?

Reply:
@dan-hill2802  - if it fails, then nothing will be running, so would rather the subsequent terraform step still runs. Will amend the PR to continue the pipeline if it passes or fails.

Reply:
Changed the exits to exit 1 on failure and added ensure step so the terraform apply step runs whether or not the cycle step passes.